convert2rhel/actions/pre_ponr_changes/custom_repos_are_valid.py

```diff
@@ -37,8 +38,8 @@ def run(self):
         super(CustomReposAreValid, self).run()
         logger.task("Prepare: Check if --enablerepo repositories are accessible")
 
-        if not tool_opts.no_rhsm:
-            logger.info("Did not perform the check of repositories due to the use of RHSM for the conversion.")
+        if not tool_opts.enablerepo:
+            logger.info("Skipping the check as there was no --enablerepo option detected in the command-line.")
```

Comment:
We changed this from orignally saying "Skipping" so we don't confuse customers with our SKIP result level. Reference - https://issues.redhat.com/browse/RHELC-1173

Reply:
```suggestion
            logger.info("Did not perform the check of repositories due to the use of RHSM for the conversion.")
```